sdks/python/README.md

```diff
@@ -339,89 +340,91 @@ text file from [Google Cloud Storage](https://cloud.google.com/storage/).
 
 ```python
 import re
-import google.cloud.dataflow as df
-p = df.Pipeline('DirectPipelineRunner')
+import apache_beam as beam
+p = beam.Pipeline('DirectPipelineRunner')
 (p
- | df.Read('read',
-           df.io.TextFileSource(
-           'gs://dataflow-samples/shakespeare/kinglear.txt'))
- | df.FlatMap('split', lambda x: re.findall(r'\w+', x))
- | df.combiners.Count.PerElement('count words')
- | df.Write('write', df.io.TextFileSink('./results')))
+ | 'read' >> beam.Read(
+        beam.io.TextFileSource('gs://dataflow-samples/shakespeare/kinglear.txt')
+    )
```

Comment:
I meant that the `|` and `)` does not look aligned. I might be seeing wrong.

Also you are right that 2 spaces is the current standard for this code base.